lib/rules/links/linkchecker.js

```diff
@@ -71,14 +71,30 @@ exports.check = async function (sr, done) {
             sr.error(compound, 'not-same-folder', { base: thisLink, url });
         }
 
-        // check if every resource's status code is ok
-        if (!response.ok() && !noRespondAllowList.includes(url)) {
-            sr.error(compound, 'response-error', {
-                url,
-                status: response.status(),
-                text: response.statusText(),
-                referer,
-            });
+        // check if every resource's status code is ok, ignore 3xx
+        if (
+            !response.ok() &&
+            response.status() >= 400 &&
```

Comment:
`ok()` will return true for code between 200 and 299 so this is redundant
```suggestion
            response.status() >= 400 &&
```